docs/MAX31855.md

```diff
@@ -0,0 +1,66 @@
+# MAX31855 / MAX6675 Thermocouple Amplifier
+
+??? failure "This feature is not included in precompiled binaries"
```

Comment:
This isn't displaying right when I "view file", I'm not sure if I have an error in the Markdown, or it's just a special feature that doesn't show in the PR view.

Reply:
markdown renderer included in editors such as VSCode are not exactly identical to the markdown renderer used in MkDocs so they may not be appear as expected in your editor while being perfectly correct.

This looks like fine to me